engines/sludge/builtin.cpp

```diff
@@ -2074,7 +2074,8 @@ builtIn(deleteFile) {
 	namNormal.clear();
 	if (failSecurityCheck(nam))
 		return BR_ERROR;
-	fun->reg.setVariable(SVT_INT, remove(nam.c_str()));
+
+	fun->reg.setVariable(SVT_INT, !g_system->getSavefileManager()->removeSavefile(nam.c_str()));
```

Comment:
```suggestion
	fun->reg.setVariable(SVT_INT, !g_system->getSavefileManager()->removeSavefile(nam));
```

Reply:
Actually, if you feel like it, it might be good to also do the change for `g_system->getSavefileManager()->renameSavefile()` a few lines below.

Reply:
Changed both instances